config/settings.yml

```diff
@@ -9,6 +9,7 @@ google_client_id:
 google_client_secret:
 google_maps_api_key:
 single_tenant: <%= ENV['SINGLE_TENANT'] %>
+public_registration: <%= ENV['PUBLIC_REGISTRATION'] %>
```

Comment:
Can we avoid the breaking change and accept an unset variable as `true`. This makes the registration feature effectively opt-out, which IMO should be okay.